docs/FOLDERS.md

```diff
@@ -0,0 +1,94 @@
+# Folders
+
++-- [api/](#api)   
+|  +-- [common/](#api/common)   
++-- [config/](#config)  
++-- [docs/](#docs)  
++-- [events/](#events)  
++-- [lib/](#lib)  
++-- [scripts/](#scripts)  
++-- [services/](#services)  
+|  +-- [models/](#services/models)   
++-- [workers/](#workers)  
++-- [.editorconfig](#.editorconfig)  
++-- [.env.sample](#.env.sample)  
++-- [.gitignore](#.gitignore)  
++-- [.npmrc](#.npmrc)  
++-- [CONTRIBUTION.md](#.CONTRIBUTION.md)  
++-- [Dockerfile](#Dockerfile)  
++-- [package.json](#package.json)  
++-- [README.md](#README.md)
+
+## api
+Contains:
+ 1. Actions, scripts that interact with the services in [services](#services) folder.
+ 2. Endpoints, exposing the service to other services.
+
+### api/common
+This folder contains all the logic that is shared across multiple files in the parent folder ([api](#api)).
+
+## config
+This is where configurations are loaded, configurations could be loaded from the environment variables either in production or in development based on where the service has been spawned.
+
+## docs
+All the docs for this service should be here.
+
+## Events
+This is where all the events handlers should be located. Events are very ideal for queue related operations.
+
+## lib
+Contains the wrappers to external libraries.
+
+## scripts
+This contains all scripts that are run once or periodically, E.g. Migration scripts, indice creators etc.
```

Comment:
not periodically, just once
one time adminstrative scripts or local test utitlites as well any build configs